payment/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.core.validators import MinValueValidator
+
+from course.models import Course, Session
+from account.models import Student
+
+from decimal import *
+
+
+class Payment(models.Model):
+    CASH = 'CASH'
+    CARD = 'CARD'
+    CHECK = 'CHECK'
+    ACCOUNT_CREDIT = 'ACCOUNT_CREDIT'
+    PAYMENT_METHOD_CHOICES = (
+        (CASH, 'Cash'),
+        (CARD, 'Card'),
+        (CHECK, 'Check'),
+        (ACCOUNT_CREDIT, 'Account Credit')
+    )
+
+    method = models.CharField(
+        max_length=14,
+        choices=PAYMENT_METHOD_CHOICES,
+    )
+    amount = models.DecimalField(max_digits=6, decimal_places=2, validators=[MinValueValidator(Decimal('0.00'))])
+    description = models.CharField(max_length=1000)
+    date_time = models.DateTimeField(auto_now=True)
```

Comment:
let's be consistent with the other models: use created_at and updated_at (with auto_now=True for updated_at and auto_now_add=True for created_at)